contracts/MemeCelo.sol

```diff
@@ -177,4 +149,6 @@ contract MemeCelo is MemeFactory {
 
         return msg.value;
     }
+
+    function _wrap(uint256) internal virtual override {}
```

Comment:
what's thsi for?

Reply:
On Celo we just don't need that as CELO and wCELO are the same token.